lib/get-exporter.js

```diff
@@ -0,0 +1,121 @@
+const paginateRequest = require('./paginate-request');
+const path   = require('path');
+const inquirer = require('inquirer');
+const {
+  loadConfig,
+  getShallowStatus,
+  getShallowComparativeStatus,
+  logResult,
+  logProcessing,
+  logError,
+  mapById,
+  plural
+} = require('./utils');
+const {
+  remove,
+  pathExists,
+  writeFile,
+  ensureDir
+} = require('fs-extra');
+const { merge, values, isEmpty } = require('omnibelt');
+const allSettledSerial = require('./all-settled-serial-p');
+const { rollbarLog } = require('./rollbar');
+
+const getExporter = ({
+  apiType, commandType, localStatusParams, remoteStatusParams, getData, extraQuery
+}) => {
+  const download = async (pattern, command = {}, loadedConfig) => {
+    let didDownload = false;
+    const { apiToken, applicationId, api } = loadedConfig ? loadedConfig : await loadConfig();
+    if (!apiToken || !applicationId) { return; }
+    let items;
+    try {
+      let query = { applicationId };
+      if (extraQuery) {
+        query = merge(query, extraQuery);
+      }
+      items = await paginateRequest(api[apiType].get, query);
+    } catch (e) {
+      return logError(e);
+    }
+    const itemsById = mapById(items);
+    const { remoteStatusByFile, localStatusByFile } = await getShallowStatus({
+      items,
+      remoteStatusParams,
+      localStatusParams,
+      dir: commandType
+    });
+    if (command.dryRun) {
+      logResult('DRY RUN');
+    }
+    if (isEmpty(remoteStatusByFile)) {
+      if (!pattern) {
+        return logResult('Missing', `No ${plural(commandType)} found to export.`, 'yellow');
+      } else {
+        return logResult('No Matches', `No ${plural(commandType)} found that match this pattern ${pattern}`, 'yellow');
+      }
+    }
+    const downloadResults = await allSettledSerial(async (remoteStatus) => {
+      logProcessing(remoteStatus.file);
+      if (!command.force) {
+        const localStatus = localStatusByFile[remoteStatus.file];
+        const { conflict } = getShallowComparativeStatus(localStatus, remoteStatus);
+        if (conflict) {
+          const { handleConflict } = await inquirer.prompt([{
+            name: 'handleConflict',
+            type: 'list',
+            message: `The file ${remoteStatus.file} already exists, how do you want to handle this?`,
+            choices: [
+              { name: 'Do nothing, and resolve the conflict later.', value: null },
+              { name: 'Overwrite with the remote data.', value: 'overwrite' },
+              { name: 'Ignore the remote data.', value: 'local' }
+            ]
+          }]);
```

Comment:
I would make this prompt a yes or no question, `The file ${remoteStatus.file} already exists, do you want to overwrite it?`